src/gateway/ws.rs

```diff
@@ -184,19 +239,19 @@ impl WsClient {
 
         debug!("[Shard {:?}] Sending presence update", shard_info);
 
-        self.send_json(&json!({
-            "op": Opcode::PresenceUpdate,
-            "d": {
-                "afk": false,
-                "since": now,
-                "status": status.name(),
-                "game": activity.as_ref().map(|x| json!({
-                    "name": x.name,
-                    "type": x.kind,
-                    "url": x.url,
-                })),
+        self.send_json(&WebSocketMessage {
+            op: Opcode::RequestGuildMembers,
```

Comment:
```suggestion
            op: Opcode::PresenceUpdate,
```
Small mistake from fixing conflicts